examples/with-solana-passkeys/package.json

```diff
@@ -0,0 +1,44 @@
+{
+  "name": "@turnkey/demo-solana-passkeys",
+  "version": "0.1.0",
+  "private": true,
+  "scripts": {
+    "dev": "next dev",
+    "build": "next build",
+    "start": "next start",
+    "lint": "next lint",
+    "typecheck": "tsc --noEmit"
+  },
+  "engines": {
+    "node": ">=18.0.0"
+  },
+  "dependencies": {
+    "@turnkey/api-key-stamper": "workspace:*",
+    "@turnkey/http": "workspace:*",
+    "@turnkey/sdk-browser": "workspace:*",
+    "@turnkey/sdk-react": "workspace:*",
+    "@turnkey/sdk-server": "workspace:*",
+    "@turnkey/solana": "workspace:*",
+    "@turnkey/webauthn-stamper": "workspace:*",
+    "axios": "^1.4.0",
+    "bs58": "^5.0.0",
+    "encoding": "^0.1.13",
+    "eslint": "8.43.0",
+    "eslint-config-next": "14.2.10",
+    "esm": "^3.2.25",
+    "install": "^0.13.0",
+    "next": "14.2.10",
+    "npm": "^9.7.2",
+    "react": "18.2.0",
+    "react-dom": "18.2.0",
+    "react-hook-form": "^7.45.1",
+    "tweetnacl": "^1.0.3",
```

Comment:
Nit: we don't use this atm - although should we? Maybe referencing the website makes more sense

Reply:
removed!